src/lifecycle/index.tsx

```diff
@@ -226,6 +253,7 @@ function reducer(previousState: State, action: Action): State {
           connectivity: { type: 'Following', endpoints },
           block: previousState.block,
           chain: previousState.chain,
+          account,
```

Comment:
I believe the account should get updated in this state instead of RestoredState to reflect the latest changes.
Does it also need to get updated on RedtoredState as well?! The reason I removed it from RestoredSate was because RestoredState type does not include account. Does this changes make sense?!